storybook/src/docs/components/ClearInputAdornment/ClearInputAdornment.mdx

```diff
@@ -6,13 +6,7 @@ import * as ClearInputAdornmentStories from "./ClearInputAdornment.stories";
 
 # ClearInputAdornment
 
-`ClearInputAdornment` adds a button to clear the value of an input field.<br />
+`ClearInputAdornment` adds a button to clear the value of an input field. It is used automatically, whenever a field is optional.<br />
```

Comment:
I'm not sure this is relevant, as the component will only be used directly when creating custom fields, where this would need to be implemented manually 🤔 

```suggestion
`ClearInputAdornment` adds a button to clear the value of an input field.<br />
```